diff --git a/jsf-tools/src/main/java/com/sun/faces/tools/StripClassesForApiJar.java b/jsf-tools/src/main/java/com/sun/faces/tools/StripClassesForApiJar.java
--- a/jsf-tools/src/main/java/com/sun/faces/tools/StripClassesForApiJar.java	2016-02-04 03:54:54.000000000 +0100
+++ b/jsf-tools/src/main/java/com/sun/faces/tools/StripClassesForApiJar.java	2026-08-02 22:56:33.874063698 +0200
@@ -49,6 +49,7 @@
 import com.sun.tools.javac.code.Symbol.*;
 import com.sun.tools.javac.code.Flags;
 import com.sun.tools.javac.code.Type;
+import com.sun.tools.javac.code.Types;
 import com.sun.tools.javac.jvm.ClassReader;
 import com.sun.tools.javac.jvm.ClassWriter;
 import com.sun.tools.javac.jvm.Pool;
@@ -182,7 +183,8 @@
         ClassReader reader = ClassReader.instance(task.getContext());
         ClassWriter writer = ClassWriter.instance(task.getContext());
         Type.moreInfo = true;
-        Pool pool = new Pool();
+        Types types = Types.instance(task.getContext());
+        Pool pool = new Pool(types);
 	ClassSymbol cs = null;
         try {
             for (JavaFileObject file : fm.list(jarLocation, "", EnumSet.of(CLASS), true)) {
