diff --git a/jsf-ri/conf/xslt/merge-config.xsl b/jsf-ri/conf/xslt/merge-config.xsl
--- a/jsf-ri/conf/xslt/merge-config.xsl	2016-02-04 03:54:54.000000000 +0100
+++ b/jsf-ri/conf/xslt/merge-config.xsl	2026-08-02 22:56:33.651838361 +0200
@@ -47,7 +47,6 @@
     <xsl:output method="xml" indent="yes"/>
 
     <xsl:strip-space elements="*"/>
-    <xsl:namespace-alias stylesheet-prefix="jsf" result-prefix="#default"/>
 
     <xsl:variable name="source" select="document('../../resources/jsf-ri-config.xml')"/>
 
